### ISSUE TITLE
allow site to request preferred email or domain

### ISSUE THREAD

**Issue by benadida** [mozilla/persona#311]
adding issue here to track discussion thread:

https://groups.google.com/group/mozilla.dev.identity/browse_thread/thread/2b750cb313f34a9/d7db3cef2b3327d7

**Comment by lloyd** [comment]
required email is added, we'll wait on 'preferred features' and open a new issue when it becomes clear that we need them.
